extensions/objects/templates/translatable-set-of-normalized-string-editor.component.ts

```diff
@@ -55,8 +55,8 @@ export class TranslatableSetOfNormalizedStringEditorComponent {
 
   getSchema(): {
     type: string;
-    items: { type: string; };
-    validators: { id: string; }[];
+    items: { type: string };
+    validators: { id: string }[];
     } {
```

Comment:
```suggestion
  } {
```

Reply:
@vojtechjelinek This suggested change and the one below it (for file `translatable-set-of-unicode-string-editor.component.ts`) cause indentation errors because the number of spaces is expected to be 4.

Additionally, I've noticed that this type is used twice in the file:
![image](https://user-images.githubusercontent.com/78612244/141748849-12d84335-fd6e-482b-b7f3-60c6342cc19d.png)

(And similarly for `translatable-set-of-unicode-string-editor.component.ts`)

I could store this type in a variable and use that instead (for both the files).

Reply:
Yes, please do that, and also feel free to indent it by 4 spaces since it is a bit more readable.

Reply:
Done